source/code-snippets/usage-examples/changeStream.js

```diff
@@ -21,21 +26,15 @@ async function run() {
       console.log("received a change to the collection: \t", next);
     });
 
-    // use a timeout to ensure the listener is registered before the insertOne
-    // operation is called.
-    await new Promise(resolve => {
-      setTimeout(async () => {
-        await movies.insertOne({
-          test: "sample movie document",
-        });
-        // wait to close `changeStream` after the listener receives the event
-        setTimeout(async () => {
-          resolve(
-            await changeStream.close(() => console.log("closed the change stream"))
-          );
-        }, 1000);
-      }, 1000);
+    await simulateAsyncPause();
+
+    await movies.insertOne({
```

Comment:
Suggestion: Could the database be changed to use the `insertDB` database? I think this is used as the[ insert database on Node insert pages](https://docs.mongodb.com/drivers/node/current/usage-examples/insertOne/#example).

Reply:
It can, but I don't see it adding additional value so I will leave it as is.